extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -798,29 +798,6 @@ static public function setVarById( $cv_variable_id, $city_id, $value, $reason=nu
 						__METHOD__ );
 					break;
 
-				case "wgDBname":
-					#--- city_dbname
-					$dbw->update(
-						static::table("city_list"),
-						[ "city_dbname" => $value ],
-						[ "city_id" => $city_id ],
-						__METHOD__ );
-					break;
-
-				case "wgDBcluster":
-					/**
-					 * city_cluster
-					 *
-					 * city_cluster = null for first cluster
-					 * @todo handle deleting values of this variable
-					 */
-					$dbw->update(
-						static::table("city_list"),
-						[ "city_cluster" => $value ],
-						[ "city_id" => $city_id ],
-						__METHOD__ );
-					break;
-
```

Comment:
These WikiFactory variables will be removed from `city_variables_pool` table when this goes live.

These `city_list` fields will be used instead:

```sql
  `city_dbname` varchar(64) NOT NULL DEFAULT 'notreal',
  `city_cluster` varchar(255) DEFAULT NULL,
```